### PR TITLE
DOC: add clarity on the purpose of npm run env and how it works

### PR DESCRIPTION
**Pull request by captaincaius**
Added @sinedied 's very well-worded description of npm run env and a little caveat to ensure that nobody gets a false sense of security when using it.

I'm not sure it's the best place for it, but at least someone will inevitably find that comment fairly early on when configuring their environments.

If you think there's a better place, or that the caveat is too obvious I'd understand :).

### CODE REVIEW

File: generators/app/templates/src/environments/environment.prod.ts [hunk 1]
```diff
@@ -1,4 +1,9 @@
 // `.env.ts` is generated by the `npm run env` command
+// Using `npm run env` is to expose environment variables as JSON for any usage you might
```
Comment (sinedied) [inline generators/app/templates/src/environments/environment.prod.ts]: "Using `npm run env` is to expose" -> "`npm run env` expose"

makes it easier to read IMO

**Comment by sinedied** [comment]
Thanks for the PR! Other than the working suggestion I noted, I'm OK with it and it's a good place to put it :)

File: generators/app/templates/src/environments/environment.prod.ts (hunk 1 above)
Comment (captaincaius) [reply generators/app/templates/src/environments/environment.prod.ts]: cool beans :) i'll reword it tonight after I eat something ;)

Comment (captaincaius) [reply generators/app/templates/src/environments/environment.prod.ts]: updated
